Sources/Sentry/SentryDebugMeta.m

```diff
@@ -23,6 +23,49 @@ - (instancetype)init
     return serializedData;
 }
 
+- (BOOL)isEqual:(id)object;
```

Comment:
M: I assume this is being used implicitly now? I can't spot a usage besides the unit tests, where is it used from?

Reply:
M: wondering if it was possible to compare field-by-field in the tests and get rid of it, as it's only used by tests
we also dont implement equals/hashcode on Android/Java

Reply:
Currently, we only use it in the tests. Wouldn't you rather have them implemented than comparing field by field in the tests, because then you can also compare them everywhere and they work properly with sets and hashtables?

Reply:
H: Can wr define them in a test file then? Having this type of code here being deployed to all devices of our customers if it's just for test is not a good approach imho

Reply:
I get your point. YAGNI. I changed it. Thanks for your valid feedback. 💯